### ISSUE TITLE
loss is so big 

### ISSUE THREAD

**Issue by Ji-qing** [xw-hu/DSC#9]
Hi, when I run train.py in DSC_detection, the loss was very big, like  this:
`Iteration 60, loss = 640045
I1116 18:20:46.603199 22386 solver.cpp:244]     Train net output #0: loss-dsn1 = 122860 (* 1 = 122860 loss)
I1116 18:20:46.603204 22386 solver.cpp:244]     Train net output #1: loss-dsn2 = 117984 (* 1 = 117984 loss)
I1116 18:20:46.603207 22386 solver.cpp:244]     Train net output #2: loss-dsn3 = 123544 (* 1 = 123544 loss)
I1116 18:20:46.603211 22386 solver.cpp:244]     Train net output #3: loss-dsn4 = 84440.4 (* 1 = 84440.4 loss)
I1116 18:20:46.603215 22386 solver.cpp:244]     Train net output #4: loss-dsn5 = 104531 (* 1 = 104531 loss)
I1116 18:20:46.603219 22386 solver.cpp:244]     Train net output #5: loss-dsn6 = 87796.2 (* 1 = 87796.2 loss)
I1116 18:20:46.603222 22386 solver.cpp:244]     Train net output #6: loss-fuse = 79751.9 (* 1 = 79751.9 loss)
I1116 18:20:46.603225 22386 solver.cpp:244]     Train net output #7: loss-global = 120688 (* 1 = 120688 loss)
`
Could you give me some suggestions? Thank you  very much!

**Comment by xw-hu** [comment]
No problem, if only the loss decreases.
